### ISSUE TITLE
DeathChest Support or Option

### ISSUE THREAD

**Issue by zrocweb** [alkarinv/BattleArena#128]
Hey.. This is by far THE BEST battle/pvp plugin EVER.  So many options, so many different ways to configure default as well as custom competitions.. Love it...  

Now that I've said that, I would love to see any option, perhaps it's there and I just missed it, for when a player dies, to drop or not drop their items.  I think that is there, but I have a unique situation.  I use DeathChest plugin for all my worlds on my server.  If a player does not have a DeathChest configured for that world when he dies his items are placed in a tombstone (bedrock/sign).  There is a way for me to turn off tombstones of the plugin but that would affect it globally across other worlds and is not the desired option.  I actually like the tombstone being placed in my arenas and players can just pillage those tombstones to get the dropped items rather than a ton of items being laid across the maps until they fade out...

However, the issue is this and I'm looking more into the DeathChest configuration to see if something is missing there, but when a player breaks a tombstone and that player is not op nor is that player the same player as the tombstone is for, nothing appears as the drops. This again, might be something of deathchest plugin and i'm looking into, but was wondering if perhaps you could add a softDepend for something like this or perhaps BA's own TombStone methods and disable other plugins that perform this functionality in at death.  Of course to disable any others, it would have to be an option in config to specify the plugins name and then perhaps onDeath event could bypass that plugins (config.death.plugin.name) onDeath event........

If I've missed anything or your not able to understand what I mean,  please let me know I will re-explain.  Thanks!!!

**Comment by alkarinv** [comment]
1) http://battleplugins.com/w/index.php/BA_Config#Clearing_Inventory_on_Death

So would the problem still be there after you clear the persons inventory on death?

  I could hook into death chest maybe, but the problem is there are a lot of plugins that affect drops after death, and I don't want to have to start hooking into all of them.

**Comment by zrocweb** [comment]
Hey. Thanks for the response. Ummm what's happens is that I have the clearinventory set on death but the plugin deathchest takes over and places a tombstone where the player dies. And noone except for op and that player can raid and the tombstone.  

Sent from galaxy III s

-------- Original message --------

**Comment by alkarinv** [comment]
does death chest have permissions?  if so I can write in a workaround.  Link me a page with their perms if they do!

**Comment by zrocweb** [comment]
Here are the perms for the DeathChest Plugin.  http://dev.bukkit.org/server-mods/deathchest/pages/permissions/

**Comment by alkarinv** [comment]
So those perms dont include a don't drop death chest :(

if they have no inventory does it still make a chest?

**Comment by zrocweb** [comment]
No. If they don't have any inv it won't do anything.  But there should be a perm in there for the sign create.  Let me look at the actual config setting and what its doing?

**Comment by alkarinv** [comment]
onDeath.options: [clearInventory], should work then to stop it.  But yeah, check it out

**Comment by zrocweb** [comment]
The [clearInventory] option does not work. I currently have it in there.   my onDeath stage looks like this:

onDeath:
   options: [respawn, clearInventory]

The perm for weather the player gets a chest or tombstone at death is: deathchest.spawnchest.
The type that is placed depends on a config setting: DeathSpawnType: chest, grave or none.
